test/Microsoft.AspNetCore.Mvc.FunctionalTests/WebApiCompatShimActionSelectionTest.cs

```diff
@@ -329,6 +327,25 @@ public WebApiCompatShimActionSelectionTest(MvcTestFixture<WebApiCompatShimWebSit
         }
 
         [Theory]
+        [InlineData("api/Admin/Test", "PostUser")]
+        [InlineData("api/Admin/Test?name=mario&age=10", "PostUserByNameAndAge")]
+        public async Task LegacyActionSelection_OverloadedAction_WithUnnamedAction(string requestUrl, string expectedActionName)
```

Comment:
Are these `LegacyActionSelection_...` tests something you added? A rebase artifact? ...